src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternUtil.java

```diff
@@ -519,6 +521,13 @@ public static String applyModifiers(String label, List<String> parts, int offset
                     }
                     resultingLabel = abbreviateSB.toString();
 
+                } else if ("camel".equals(modifier)) {
```

Comment:
Maybe. I still wonder how many of those are really useful and how many of them may lead to unexpected problems... We would in that case still need to have some explicit name mapping to not loose compatibility, introduce the (rather useless in other cases) CamelCaseFormatter, and come up with a new syntax. I did in fact consider it earlier, but didn't really see the benefit of it.